docs/cookbook/modifying-a-template-in-administration.md

```diff
@@ -29,6 +29,9 @@ See especially the new directory `ShopsysFrameworkBundle` whose title is based o
 Thanks to this exact location, your new copy of the template will be used instead of the original template from the FrameworkBundle during the rendering process.
 At this point, you just need to modify your copy of the template in such a way that product transfer status will be displayed on the page.
 
+!!! note
+    If you want to change only some block you can override the original template. For that, you need to use `extends` macro with an exclamation mark to prevent template cycling. For example: `{% extends '@!ShopsysFramework/Admin/Content/Product/detail.html.twig' %}`
```

Comment:
I think that better would be to direct user to [symfony docs](https://symfony.com/doc/current/bundles/override.html#templates)

Reply:
Perhaps we can combine both. Lets just add link to symfony doc at the and of henzigo's cookbook. Something like "More information can be found in [official symfony documentation](https://symfony.com/doc/current/bundles/override.html#templates)"

Reply:
I like the idea of @sslt. I upgraded docs and rebased to master